src/comfywatchman/config.py

```
@@ -150,18 +150,56 @@ def state_dir(self) -> Path:
+        Args:
+            config_file: Path to the TOML configuration file
+
+        Returns:
+            bool: True if loaded successfully, False otherwise
+        """
         if config_file.exists():
             try:
                 with open(config_file, "rb") as f:
                     toml_data = tomllib.load(f)
                 self._update_from_dict(self, toml_data)
+                return True
```

Ruff (0.15.14)

[warning] Consider moving this statement to an `else` block

(TRY300)